include/state-observation/tools/rigid-body-kinematics.hxx

```diff
@@ -1751,6 +1904,125 @@ inline Kinematics Kinematics::setToProductNoAlias(const Kinematics & multiplier1
   return *this;
 }
 
+inline Kinematics Kinematics::zeroKinematics(Flags::Byte flags)
+{
+  Kinematics kine;
+  kine.setZero(flags);
+  return kine;
+}
+
+inline Kinematics Kinematics::setToDiffNoAlias(const Kinematics & multiplier1, const Kinematics & multiplier2)
```

Comment:
I am not a big fan of early returns in general, but I will consider this case :)

Reply:
Implemented in commit 607a7779bd0a9c80b54da8accd8e9486cd04f44d: split setToDiffNoAlias into lin and ang parts
I noticed you rewrote some expressions but I kept the old ones because the results were not correct using yours (sorry).